examples/crate_universe/multi_package/sub_pkgs/pkg_b/Cargo.toml

```diff
@@ -6,5 +6,4 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rustls = "0.20.8"
-rustls-pemfile = "1.0.2"
+rustls = "0.21.12"
```

Comment:
Was there something that drove this change? It seems like a slightly big thing to be coupling with a version update (but I don't mind, just curious :))

Reply:
I ran into this failure so opted to simplify the change. The bump to rusttls wasn't strictly necessary though
```
==================== Test output for //multi_package/sub_pkgs/pkg_b:unit_test:

running 1 test
test test::client_verifier_works ... FAILED

failures:

---- test::client_verifier_works stdout ----
thread 'test::client_verifier_works' panicked at multi_package/sub_pkgs/pkg_b/src/lib.rs:177:9:
assertion `left == right` failed
  left: "Err(Client(InvalidCertificateData(\"invalid peer certificate: CertExpired\")))"
 right: "Ok(())"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    test::client_verifier_works

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```